### ISSUE TITLE
Badges have disappeared?

### ISSUE THREAD

**Issue by akshaaatt** [pavi2410/PlayBadges#13]
Can we fix this asap?

**Comment by pavi2410** [comment]
it's working again. thanks for reporting 👍

**Comment by akshaaatt** [comment]
@pavi2410 seems like the issue has surfaced again. Can you please have a look at it?

**Comment by pavi2410** [comment]
Yep, I found out yesterday that the Play Store website has been rewritten. So the scraper must be rewritten as well. I will try to resolve that ASAP

**Comment by akshaaatt** [comment]
Thanks!

**Comment by pavi2410** [comment]
It's back up online!

**Comment by akshaaatt** [comment]
Thank you!

**Comment by armandojimenez** [comment]
Hi! Is not working again!
